data/wanderer ships.txt

```diff
@@ -728,3 +728,57 @@ ship "Deep River Transport"
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
 	description "This is a Deep River freighter that has been converted into a transport ship for refugees."
+	
+ship "Riptide"
+    sprite "ship/riptide"
```

Comment:
These are all spaces, not tabs. The only line that isn't spaced in instead of tabbed in is the `"Wanderer Heat Sink"` outfit.